vercel/turborepo
# feat(examples): update next.js 15 and eslint 9

### Description

This PR is a work in progress. The goal is to migrate the examples to use Next.js 15 with Eslint 9 config format.

I've decided to decouple the examples from `@vercel/style-guide` since it doesn't use much of that package.

### Testing Instructions

Run `pnpm lint` in the basic example. It should show everything in green. Add a `debugger` statement in one of the apps or ui package and it should display a lint error.
